Documentation/api.md

```diff
@@ -143,6 +143,7 @@ AlertmanagerSpec is a specification of the desired behavior of the Alertmanager
 | containers | Containers allows injecting additional containers. This is meant to allow adding an authentication proxy to an Alertmanager pod. | []v1.Container | false |
 | priorityClassName | Priority class assigned to the Pods | string | false |
 | additionalPeers | AdditionalPeers allows injecting a set of additional Alertmanagers to peer with to form a highly available cluster. | []string | false |
+| portName | Port name used for the pods and governing service This defaults to web | string | false |
```

Comment:
`This` -> `this`

Reply:
Ah, this is from a two line comment in the api definition. Will fix now

Reply:
Fixed! :)